Directory.Build.props

```diff
@@ -15,17 +15,17 @@
     <TargetFramework>net6.0</TargetFramework>
     <NoWarn>$(NoWarn);NU5125;CA1416;NU1507</NoWarn>
     <!--
-      We sort of half sort of support macOS in this repo, this warning is telling us that we are on the precipice of 
+      We sort of half sort of support macOS in this repo, this warning is telling us that we are on the precipice of
```

Comment:
"we sort of half sort of" is really awkward. i only mention it because you appear to be fixing this line anyways... :D

Reply:
I wonder if I wrote that... it sounds like something I would write :D

Reply:
VS Code just changed this line to remove trailing whitespace. Feel free to attempt to parse the sentence and create an issue improving it &mdash; separately :grin: